std/testing/asserts.ts

```diff
@@ -329,6 +329,23 @@ export function assertNotStrictEquals(
   );
 }
 
+/**
+ * Make an assertion that actual is not null or undefined. If not
+ * then thrown.
+ */
+export function assertAnything(
+  actual: unknown,
+  msg?: string,
+): void {
```

Comment:
It seems you can already achieve the same assertion using just `assert()`. Is there some edge case not covered by it?

Reply:
`assert(false)` would throw, while `assertAnything(false)` would not.

Reply:
I prefer explicit assertions, if you're checking that something is not null and not undefined there's probably a very specific reason for that.

```typescript
assert(value !== undefined && value !== null);
```